Spec/SpecSpec.mm

```diff
@@ -205,7 +205,6 @@ void expectFailure(CDRSpecBlock block) {
             globalValue__ = @"something";
         });
 
-        expect(globalValue__).to(be_nil);
```

Comment:
@briancroom I think this was added in error 2 years ago... What's odd is that the example further up on line 174 seems to imply this should have raised an exception in the main spec run, but it only occurred when the spec gets redefined via construction of a `CDRSpecRun` in `CDRHooksSpec`.